packages/config/packages/config/config/devices/0x0371/zw141.json

```diff
@@ -0,0 +1,503 @@
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Default",
+					"value": 0
+				},
+				{
+					"label": "Confirm calibration",
+					"value": 1
+				}
+			]
+		},
+		"38": {
+			"label": "Calibration mode status report",
+			"description": "Get calibration mode status report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Calibration mode complete",
+					"value": 0
+				},
+				{
+					"label": "Start calibration mode, going to reference point A",
+					"value": 1
+				},
+				{
+					"label": "Reach reference point A, going to reference point B",
+					"value": 2
+				},
+				{
+					"label": "Reach reference point B, waiting for blade turn calibration",
+					"value": 3
+				},
+				{
+					"label": "Going to reference point C",
+					"value": 4
+				},
+				{
+					"label": "Calibration mode terminated",
+					"value": 5
+				}
+			]
+		},
+		"39": {
+			"label": "Curtain mode opeation",
+			"description": "Set the curtain mode operation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Shutter mode",
+					"value": 0
+				},
+				{
+					"label": "Venetian mode",
+					"value": 1
+				}
+			]
+		},
+		"40": {
+			"label": "Repositioning",
+			"description": "Set to begin repositioning",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Default",
+					"value": 0
+				},
+				{
+					"label": "In repositioning",
+					"value": 1
+				}
+			]
+		},
+		"41": {
+			"label": "Repositioning status report",
+			"description": "Get repositioning status report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Repositioning complete",
+					"value": 0
+				},
+				{
+					"label": "In repositioning",
+					"value": 1
+				}
+			]
+		},
+		"42": {
+			"label": "Calibration lock",
+			"description": "Enable/disable calibration lock",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable calibration",
+					"value": 0
+				},
+				{
+					"label": "Enable calibration",
+					"value": 1
+				}
+			]
+		},
+		"48": {
+			"label": "S1 scene activation",
```

Comment:
Change this to S1 Scene Activation Mode. Then change the option values to just "Key pressed one time," etc. Do the same for S2.